code/modules/mob/dead/observer/observer.dm

```diff
@@ -184,7 +184,7 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 	if(P)
 		if(!P.control_computer)
 			P.find_control_computer(urgent=1)
-		if(P.control_computer)
+		if(P.control_computer || istype(P, /obj/machinery/cryopod/robot))

```

Comment:
oh
the problem seems more like the robot cryo station lacks a control computer.